usecases/traverser/near_params_vector.go

```diff
@@ -95,7 +95,7 @@ func (v *nearParamsVector) vectorFromParams(ctx context.Context,
 	}
 
 	if nearVector != nil {
-		return nearVector.VectorPerTarget[targetVector], nil
+		return nearVector.Vectors[index], nil
```

Comment:
Should we check the length here to avoid a potential panic?

Reply:
don't think this can ever happen, but added a check